app/bootstrap.py

```diff
@@ -4,15 +4,30 @@
 from dotenv import load_dotenv
 
 from app.application.service import IBMDashboardService
-from app.infrastructure.object_storage.google_cloud_object_storage import GoogleCloudStorage
+from app.infrastructure import GoogleCloudStorage, PostgreSQLUserRepository,\
+    BcryptEncrypter, JWTManager
+
+JWT_ALGORITHM = "HS256"
+JWT_SECRET_KEY = "secret"
```

Comment:
Is the JWT_SECRET_KEY intended to the repo as public?

Reply:
Already added those as env variables